src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListCommand.cs

```diff
@@ -160,13 +162,15 @@ private void MenuItemCallback(object sender, EventArgs e)
                     break;
 
                 case UsefulResultCommandId:
+                    SarifErrorListItem currentItem = selectedItems.FirstOrDefault();
```

Comment:
>SarifErrorListItem currentItem = selectedItems.FirstOrDefault(); [](start = 20, length = 64)

we could remove this, since we are not using anymore.